src/EditorFeatures/Core/Diagnostics/AbstractAggregateDiagnosticsTaggerProvider.AggregateTagger.cs

```diff
@@ -1,58 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using Microsoft.CodeAnalysis.PooledObjects;
-using Microsoft.VisualStudio.Text;
-using Microsoft.VisualStudio.Text.Tagging;
-
-namespace Microsoft.CodeAnalysis.Diagnostics;
-
-internal abstract partial class AbstractAggregateDiagnosticsTaggerProvider<TTag> where TTag : ITag
-{
-    /// <summary>
-    /// Simple tagger that aggregates the underlying syntax/semantic compiler/analyzer taggers and presents them as
-    /// a single event source and source of tags.
-    /// </summary>
-    private sealed class AggregateTagger : ITagger<TTag>, IDisposable
```

Comment:
this type entirely moved.  but the nesting changed, so github didn't pick it up.  it is UNCHANGED.